ui-particles-angular/projects/ui-particles-angular/src/lib/gio-top-bar/gio-top-bar.component.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+import { GioTopBarProductTypes } from './gio-top-bar-product/gio-top-bar-product.component';
+
+@Component({
+  selector: 'gio-top-bar',
+  templateUrl: './gio-top-bar.component.html',
+  styleUrls: ['./gio-top-bar.component.scss'],
+})
+export class GioTopBarComponent {
+  @Input() public type: GioTopBarProductTypes = 'am';
```

Comment:
I don't really understand why we have the `type` and the component extends 🤔 

I will propose either one or the other.
If we have one type then we don't need to make one component per product. 
And if we want a component per product then no type we use the product component directly in the html (which would avoid the product component loading the code of another product)